src/storage/btree.c

```diff
@@ -16151,6 +16338,7 @@ btree_find_next_index_record_holding_current_helper (THREAD_ENTRY * thread_p, BT
 	      goto exit_on_error;
 	    }
 
+	  COMMON_PREFIX_PAGE_SIZE_RESET (bts);
```

Comment:
Should this code be placed right after pgbuf_fix?

Reply:
How about adding a function that wraps pgbuf_fix and pgbuf_unfix to perform COMMON_PREFIX_PAGE_SIZE_RESET?

Reply:
I wanted to do that too.
But it wasn't simple.

Reply:
Wouldn't it be simple to use it selectively only within the btree functions?
Even if it is only used in currently changed code, it would be good.

Reply:
I think it is better to call `COMMON_PREFIX_PAGE_SIZE_RESET` after `bts->C_page = NULL; (line: 16318)`.

Reply:
The purpose is to express my opinion, so if you read it, please resolve it.